### PR TITLE
Flexibus/#379/teams view as tiles

### LINKED ISSUE #379: Teams view as tiles
**#380 depends on this for tile component**
**#374 depends on this for tile component and team logic**

New Layout for "Teams" Page:

Header and footer are out of scope.
If the team has a team-picture, then tile has this picture. If not, then we use 1 of 2 default pictures (they can be easily found by link in Flexibus channel)

In wide-screens every tile has a 25% of width with paddings/margins.
If width for content is <=800px, then it's only 2 blocks in a row. (**this is not shown in pictures**)
In it is <=400 px, then only 1 block in a row. (IT IS ONLY EXAMPLE - the most important thing, that it looks good on every screen 300 to 1600px).
Don't forget that content has max-width:1200px!

- [ ] create a reusable component for the tiles that can be used in #380.

- whole tile should be clickable
- no priority for teams
- 'Deine Teams' only shows teams that user is a member of
- 'Deine Teams' is subset of 'Alle Teams' (not disjunct)

- 'Deine Teams' block is not shown when there is no data

### PR DESCRIPTION
**Pull request by emanuel-metzenthin**
Closes #379.

### CODE REVIEW

File: app/assets/stylesheets/tile.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+@media screen and (min-width : 800px) {
+    .tile-wrapper {
+        display: grid;
```
Comment (usame-algan) [inline app/assets/stylesheets/tile.css:3]: How do we handle browser support? See https://caniuse.com/#feat=css-grid
Do we have a fallback to flexbox or something similar?

File: app/views/application/_tile_template.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div onclick="location.href='<%= destination_path%>';" class="tile">
```
Comment (usame-algan) [inline app/views/application/_tile_template.html.erb]: This won't work if Javascript is disabled in the Browser. Could use an `<a>` Tag with absolute positioning to achieve the same result.

File: app/assets/stylesheets/tile.css (hunk 1 above)
Comment (emanuel-metzenthin) [reply app/assets/stylesheets/tile.css:3]: Grid is not compatible with Opera Mini and UC Browser for Android. All major browsers support it. So I guess we're fine :).

**Review by usame-algan — CHANGES_REQUESTED** [review]

File: app/views/application/_tile_template.html.erb (hunk 2 above)
Comment (emanuel-metzenthin) [reply app/views/application/_tile_template.html.erb]: I think many features are dependent of JS already ;). Nonetheless I changed it,  good point. Could you accept?

**Review by usame-algan — APPROVED** [review]

**Pull request merged**